src/components/ScreenshotDialog.vue

```diff
@@ -3,10 +3,13 @@
         <div class="modal-dialog modal-xl modal-dialog-centered">
             <div class="modal-content">
                 <div class="modal-header">
+                    <h5 id="exampleModalLabel" class="modal-title">
+                        {{ $t("Browser Screenshot") }}
```

Comment:
Please add this to https://github.com/louislam/uptime-kuma/blob/master/src/lang/en.json as otherwise this won't be avaliable for translation